.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
```

```suggestion
    directories:  # Location of package manifests
      - "/"
      - "/actions/comment"
      - "/actions/receive"
```
We need to explicitly add the non-standard dirs otherwise they will not be checked (test with dependabot cli)